src/inverseRobot.js

```diff
@@ -8,6 +8,18 @@
 
 function inverseRobot(robot) {
   // write code here
+  let newRobot = {};
+
+  for (const detail in robot) {
+    if (newRobot.hasOwnProperty(robot[detail])) {
+      newRobot = null;
+      break;
+    } else {
+      newRobot[robot[detail]] = detail;
+    }
```

Comment:
```suggestion
    if (newRobot.hasOwnProperty(robot[detail])) {
      return null;
    }
    
    newRobot[robot[detail]] = detail;
```
if you will return from function - loop will break too